docs/docs/contribute/docs/installation.md

```diff
@@ -43,54 +43,72 @@ And choose to disable the Path Length Limit (just in case your default installat
 
 Retain the other default installations settings.
 
-To confirm Python was installed successfully type either Python or Py in the windows command prompt:
+To confirm Python was installed successfully type either Python or Py in the windows command prompt. If Python is Installed it will show the version Details.
 
-> Python
+=== "Command"
+    ```
```

Comment:
It's good to leave an empty line before and after the code block. It doesn't change anything during generation but it makes the original file easier to read.

Reply:
=== "Command"

        ```
        Python
        ```

    === "Result"

Reply:
Agreed. Fixed it everywhere applicable.